Composer/packages/client/src/recoilModel/undo/history.ts

```diff
@@ -33,6 +35,15 @@ export const undoFunctionState = atomFamily<IUndoRedo, string>({
   dangerouslyAllowMutability: true,
 });
 
+export const undoStatusState = atomFamily<{ canUndo: boolean; canRedo: boolean }, string>({
+  key: 'undoStatus',
+  default: {
+    canRedo: false,
```

Comment:
Can we just create two botstate  canUndo and canRedo. The dangerouslyAllowMutability seems risky for changes that a component is subscribing to. If we move it to botstates, history.ts  can just set it and other components can subscribe to it